server/src/main/java/org/apache/druid/segment/realtime/appenderator/AppenderatorImpl.java

```diff
@@ -332,6 +332,47 @@ public AppenderatorAddResult add(
       if (allowIncrementalPersists) {
         // persistAll clears rowsCurrentlyInMemory, no need to update it.
         log.info("Flushing in-memory data to disk because %s.", String.join(",", persistReasons));
+
+        long bytesPersisted = 0L;
```

Comment:
super nitpick, but this should probably be named `bytesToBePersisted` or something since it isn't actually persisted yet

Reply:
To avoid unnecessary cycle on Travis, I'm merging this in for now. However, I do agree that `bytesToBePersisted` is a better name for the variable. I'll sneak this change in some other PR since it's very small/trivial ;)